src/test/java/org/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -183,6 +183,28 @@ public void localizationParameterMustIncludeAString() throws IOException {
         }
     }
 
+    @Test
+    public void localisationTestForInvalidStrings() throws IOException {
+        for (String bundle : Arrays.asList("JabRef", "Menu")) {
+            for (String lang : Languages.LANGUAGES.values()) {
+                String propertyFilePath = String.format("/l10n/%s_%s.properties", bundle, lang);
+
+                // read in
+                Properties textKeys = LocalizationParser
+                        .getProperties(propertyFilePath);
+                //parse object "textKeys" to find any spaces
+                for (Map.Entry<Object, Object> entry : textKeys.entrySet()) {
+                    String expectedKeyEqualsKey = String.format("%s=%s", entry.getKey(), entry.getKey());
+                    String actualKeyEqualsValue = String.format("%s=%s", entry.getKey(),
```

Comment:
This is not correct. While in the english localization the key must equal the value, in all other languages the the key is different from the value.